docs/implementation/fhir-data-model.md

```diff
@@ -4,7 +4,7 @@ In this section, the representation of FHIR data is discussed. In Blaze there ar
 
 ## External Serialization at the REST API
 
-Blaze supports JSON and XML serialization with JSON as default. Blaze uses [jsonista][1] to parse and generate JSON and [Clojure data.xml][2] to parse and generate XML. 
+Blaze supports JSON and XML serialization with JSON as default. Blaze uses [jsonista][1] to parse and generate JSON and [Clojure data.xml][2] to parse and generate XML.
```

Comment:
In the JSON patient, the name was missing.

Reply:
That's part of #2119.

Reply:
I separated that from the formatting itself.